doc/content/htgr/generic-pbr-tutorial/step7.md

```diff
@@ -0,0 +1,83 @@
+## Step 7
+
+In Step 7, we add the bypass flow through the control rod. The flow splits at the upper
+plenum into the portion that flows through the bed and a small bypass flow portion (a few percent)
+that flows through the control rod. This increases pebble bed temperatures slightly, but ensures that
+the control rod is cooled during reactor operations.
+
+## Modifying the Geometry
+
+A portion of the side reflector between the riser and pebble bed is replaced by the `control_rods` block (block 8). The modification that needs to be made in the `cartesian_mesh` object is simple. Some of the numbers that represents the side reflector are replaced by block number 8. We assume that the control rods are not inserted
+in this model so the porous medium contains only flow volume and graphite. The geometry is depicted in [step7mesh].
+
+Note, the `control_rods` block defines both flow variables and
+`T_solid` and the corresponding block definitions must be updated.
+
+!media generic-pbr-tutorial/MeshP7.png
+        style=width:100%
+        id=step7mesh
+        caption=Geometry for Step 7 with the fluid domain framed in red.
+
+## Updated Parameters
+
+The `thermal_mass_scaling` is reduced to obtain faster convergence
+to steady-state during the pseudo-transient. The hydraulic diameter of
+the control rods is set to `control_rod_Dh = 0.1`.
```

Comment:
maybe say how this value is determined?
is this the actual Dh of the rod or is there volume averaging involved